### DISCUSSION TITLE
Apt-key deprecated

### DISCUSSION THREAD

**Discussion by ctuxboy** [helix-editor/helix#8869] [General] [answered]
Hello,

Trying install Helix for the first time on my Chromebook (Crostini). This is a debian/ubuntu based linux distro, so i can using apt.
But when trying adding the repo it is deprecated, so i can't install Helix this way.

![image](https://github.com/helix-editor/helix/assets/10013632/5e69c603-6dda-4584-93f6-10659c1d7300)

**Comment by pascalkuthe** [comment, marked as answer]
We don't maintain these repos. They may be out of date. We can't do anything about that. I would recommend using the official release binary or building from source

**Reply by ctuxboy** [reply]
Thanks for the info 😉
